src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -1685,5 +1685,27 @@ sap.ui.define([
 		return this.bSecureSearch;
 	};
 
+	/**
+	 * Determines the next link url should be used
+	 *
+	 * This method might be overridden by the application to provide a customized next link processing because FHIR did not offer a standardized link structure.
+	 * @param {string} sNextLinkUrl The next link url
+	 * @param {string} sPath The FHIR resource path
+	 * @param {sap.fhir.model.r4.FHIRListBinding.Parameter | sap.fhir.model.r4.FHIRTreeBinding.Parameter} mParameters Existing parameters
+	 * @returns {object} Next link object containing the url and parameters
```

Comment:
"@returns {object} Next link object containing the url and parameters" Thats also a wonderful candidate for an own typedef e.g. `sap.fhir.model.r4.NextLink`.

Reply:
done